_inc/client/setup-wizard/income-question/index.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ChecklistAnswer } from '../checklist-answer';
+import Button from 'components/button';
+import { imagePath } from 'constants/urls';
+
+import './style.scss';
+
+const IncomeQuestion = props => {
+	return (
+		<div className="jp-setup-wizard-main">
+			<img
+				width="200px"
+				height="200px"
+				src={ imagePath + '/generating-cash.svg' }
```

Comment:
I get a broken image here; removing the `/` seems to fix this.

```suggestion
				src={ imagePath + 'generating-cash.svg' }
```